### ISSUE TITLE
Crash when configuring station mode

### ISSUE THREAD

**Issue by juliendarrah** [FlipFlopHome/flipflop_sming#1]
A crash occurs when trying to configure a new network when a previous SSID was configured.
The module keep trying to connect to that network and it seems to interfere with the configuration of a new network.

**Comment by juliendarrah** [comment]
Just did some more testing on the issue and it seems like when the response page is sent, the crash occurs. So I tried with a static page and I increased the size slowly till I get a crash.
Not sure how to fix this issue right now. A quick and dirty way would be to limit the number of network displayed in the list to reduce the response page size. But I'm not confident for the long term...

**Comment by ishotjr** [comment]
@juliendarrah anything I can do to help test? :bowtie:

**Comment by juliendarrah** [comment]
Yes, Try the binary I created yesterday and see if it works for you?

**Comment by juliendarrah** [comment]
Need to try this function system_get_free_heap_size() to see if the heap melts away...

**Comment by ishotjr** [comment]
@juliendarrah Apologies it took me so long to get to this; mine just connects to the existing network - can you clarify steps to set up new? :bowtie: :mag_right: :signal_strength:
